docs/components/arm/_index.md

```diff
@@ -26,6 +26,14 @@ When controlling an arm with `viam-server`, the following features are implement
 - Self-collision prevention
 - Obstacle avoidance
 
+## Related Services
+
+{{< cards >}}
+{{< relatedcard link="/services/base-rc/" >}}
```

Comment:
```suggestion
{{< relatedcard link="/services/motion" >}}
```
I don't think Base RC makes sense here, that service is better suited to input controller or base!

Reply:
This was definitely unintentional, thank you!